### DISCUSSION TITLE
update values

### DISCUSSION THREAD

**Discussion by Lukasgw** [deta/cloud-docs#307] [Help]
I tried to update my database values with the command:

```py
def update_id(updates, username):
         return db.update(updates, username)

update_id({"identity":identity}, 'test_user_key')
```

But I cannot see the changes in the Base UI, what am I doing wrong?

**Comment by abdelhai** [comment]
hi, did you refresh the ui/ click on "fetch"? the ui doesn't apply automatically

**Reply by Lukasgw** [reply]
Thanks :-)
